corehq/apps/users/tests/retire.py

```diff
@@ -197,3 +199,72 @@ def test_multiple_case_blocks_some_deleted(self, rebuild_case):
 
         self.assertEqual(rebuild_case.call_count, len(case_ids) - 1)
         self.assertItemsEqual(rebuild_case.call_args_list, expected_call_args)
+
+    @run_with_all_backends
+    def test_all_case_forms_deleted(self):
+        domain = create_domain(self.domain)
+        self.addCleanup(domain.delete)
```

Comment:
nit: `create_domain` and `domain.delete` are brutal. would be better to have this in a setupclass if possible (and this will be called twice)